### PR TITLE
feat(profiling):  add cardinality option for profiles metrics

### PR DESCRIPTION
**Pull request by viglia**
add `sentry-metrics.cardinality-limiter.limits.profiles.per-org` option and register the cardinality limit quota

### CODE REVIEW

File: src/sentry/sentry_metrics/use_case_id_registry.py [hunk 1]
```diff
@@ -45,6 +45,7 @@ class UseCaseID(Enum):
     UseCaseID.SESSIONS: "sentry-metrics.cardinality-limiter.limits.releasehealth.per-org",
     UseCaseID.SPANS: "sentry-metrics.cardinality-limiter.limits.spans.per-org",
     UseCaseID.CUSTOM: "sentry-metrics.cardinality-limiter.limits.custom.per-org",
+    UseCaseID.PROFILES: "sentry-metrics.cardinality-limiter.limits.profiles.per-org",
```
Comment (Zylphrex) [inline src/sentry/sentry_metrics/use_case_id_registry.py:48]: Can you explain what this actually limits? And what the impact will be?

Comment (Dav1dde) [reply src/sentry/sentry_metrics/use_case_id_registry.py:48]: > number of metrics

Just to clarify a bit, not metrics but distinct time series in storage.

**Review by Dav1dde — COMMENTED** [review]
One of the Project Config tests should be failing because of the newly added namespace, if it does, then the change is working.

File: src/sentry/sentry_metrics/use_case_id_registry.py (hunk 1 above)
Comment (Zylphrex) [reply src/sentry/sentry_metrics/use_case_id_registry.py:48]: Do we have any way of checking if we've exceeded this limit? And debugging why it's being exceeded.

Comment (Dav1dde) [reply src/sentry/sentry_metrics/use_case_id_registry.py:48]: @Zylphrex Yes!

We have currently 2 cardinality limiters, the Python one is disabled though. Relay has some metrics but also emits a Sentry error (sampled) for additional details.

**Review by evanh — APPROVED** [review]

**Review by Dav1dde — APPROVED** [review]

**Pull request merged**